qc_schema/dev/properties/scf_properties.py

```diff
@@ -0,0 +1,71 @@
+"""
+The complete list of SCF level properties.
+"""
+
+scf_properties = {}
+
+scf_properties["scf_one_electron_energy"] = {
+    "description": "The one-electron energy contribution to the total SCF energy.",
+    "type": "number"
+}
```

Comment:
If this is meant to refer to Hartree--Fock, not HF-or-DFT, I suggest we be explicit now and use "HF".

Reply:
What if we define that is as simply `\sum{D * H}`? I worry that splitting HF and SCF into two parts could get much more complex.